src/lazyload.js

```diff
@@ -267,6 +280,10 @@
         if (elementsLength === 0) {
             this._stopScrollHandler();
         }
+        /* Set first load variable to false */
+        if (_isFirstLoop) {
+            _isFirstLoop = false;
```

Comment:
the _isFirstLoop variable cannot be a local variable shared with all instances of lazyload, it must be an instance variable instead. So it should be referenced as this._isFirstLoop here.